modules/api/go.mod

```diff
@@ -70,7 +70,7 @@ require (
 	google.golang.org/api v0.187.0
 	gopkg.in/yaml.v3 v3.0.1
 	k8c.io/kubeone v1.7.3
-	k8c.io/kubermatic/v2 v2.25.1-0.20240823163015-db36dbe1818f
+	k8c.io/kubermatic/v2 v2.25.1-0.20240828091320-045f90ef8db4
```

Comment:
This PR doesn't "This PR will update the MC package" so please remove that from title and PR description.